build.rs

```diff
@@ -111,6 +111,11 @@ fn mm_version() -> String {
 
 fn main() {
     // This allows build script to run even if no source code files change as rerun-if-changed checks for a file that doesn't exist
```

Comment:
This comment should be moved to be above the `println!("cargo:rerun-if-changed=NON_EXISTING_FILE");` line

Reply:
Thanks! I will do it and also one more change to make failing jobs restart faster: as of now we update the MM_VERSION file every time, but if it's already up to date, we can avoid it. With this change, the failing job restart won't need to recompile at all saving up to 10 minutes depending on CI agent performance.